Makefile

```diff
@@ -20,6 +20,7 @@ export LUA_PATH
         defn defn-ocaml defn-java defn-haskell defn-llvm \
         build build-ocaml defn-haskell build-haskell build-llvm \
         test test-execution test-simple test-prove test-klab-prove \
+				test-conformance test-conformance-parse test-conformance-supported \
```

Comment:
let's use the same spacing as the line before

Reply:
whoops, looks like tabs vs. spaces. fixed